CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+#Contributing to Okta Open Source Repos
+
+## Style
+
+### Git Commit Messages
+
+We use a simplified form of [Atom's](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#git-commit-messages) commit convention.
+
+  * Use the present tense ("Adds feature" not "Added feature")
+  * Limit the first line to 72 characters or less
+  * Add one feature per commit. If you have multiple features, have multiple commits.
+  
+####Template
+
+    <emoji> Short Description of Commit
+    <BLANKLINE>
+    More detailed description of commit
+    <BLANKLINE>
+    Resolves: <Jira # or Issue #>
+
+#### Emoji Categories
+Our categories include:
```

Comment:
nit, can we order this based on what we think will be the most used? I.e. feature, bug, others